### PR TITLE
Istio 1.6.9 release notes

### PR DESCRIPTION
**Pull request by jacob-delgado**
Release notes for 1.6.9

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.6.9
+linktitle: 1.6.89
+subtitle: Patch Release
+description: Istio 1.6.9 patch release.
+publishdate: 2020-09-00
```
Comment (ericvn) [inline content/en/news/releases/1.6.x/announcing-1.6.9/index.md:6]: Obviously, we'll need the correct date.

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.6.9
+linktitle: 1.6.89
```
Comment (ericvn) [inline content/en/news/releases/1.6.x/announcing-1.6.9/index.md:3]: Change .89 to .9

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.8 and Istio 1.6.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** istioctl analyzer to detect when Destination Rules do not specify `caCertificates` ([Istio 25652](https://github.com/istio/istio/issues/25652))
```
Comment (ericvn) [inline content/en/news/releases/1.6.x/announcing-1.6.9/index.md:19]: I believe the new standard is to group the changes by verbs in an alphabetical order (don't need to change order with a given verb).

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.6.x/announcing-1.6.9/index.md]: changed

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md (hunk 2 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.6.x/announcing-1.6.9/index.md]: changed

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md (hunk 3 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.6.x/announcing-1.6.9/index.md:19]: Fixed. Thanks. Didn't know this was the new standard. I like it.

File: content/en/news/releases/1.6.x/announcing-1.6.9/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- **Added** missing `telemetry.loadshedding.*` options to mixer container arguments
+- **Fixed** HTTP match request without headers conflict
+- **Fixed** Istio operator to watch multiple namespaces ([Istio 26317](https://github.com/istio/istio/issues/26317))
```
Comment (brian-avery) [inline content/en/news/releases/1.6.x/announcing-1.6.9/index.md:22]: Nit: should these be issue #... Rather than Istio ...?

Comment (jacob-delgado) [reply content/en/news/releases/1.6.x/announcing-1.6.9/index.md]: updated

**Review by ericvn — APPROVED** [review]

**Pull request merged**